scss/components/_panel-transparent-header.scss

```diff
@@ -16,9 +16,9 @@
       margin: 0 5px;   
     }
 
-    .panel-title-inner {
-      margin: 0 10px 0 0;
-    }
+    .civicase__case-card__activity-count:nth-child(2) {
```

Comment:
This should not be a part of `_panel-transparent-header.scss`. It should be mentioned in the context of `Tasks Panel`